### PR TITLE
negar/hide_chart

### PR DESCRIPTION
**Pull request by negar-binary**
hide chart if no chart available

### CODE REVIEW

File: src/javascript/binary/pages/trade/common.js [hunk 1]
```diff
@@ -899,24 +899,28 @@ function showHighchart(){
...
-                  '</option><option value="10m">10 ' + Content.localize().textDurationMinutes.toLowerCase() + '</option><option value="15m">15 ' +
-                  Content.localize().textDurationMinutes.toLowerCase() +'</option><option value="30m">30 ' + Content.localize().textDurationMinutes.toLowerCase() +
-                  '</option><option value="1h">1 ' + text.localize('hour').toLowerCase() + '</option><option value="2h">2 ' +
-                  Content.localize().textDurationHours.toLowerCase() +'</option><option value="4h">4 ' + Content.localize().textDurationHours.toLowerCase() +
-                  '</option><option value="8h">8 ' + Content.localize().textDurationHours.toLowerCase() + '</option><option value="1d">1 ' +
-                  text.localize('day').toLowerCase() +'</option></select></td></td></tr><tr align="center"><td colspan="4">' +
-                  '<iframe src="" width="100%" height="520" id="chart_frame" style="overflow-y : hidden;" scrolling="no"></iframe></td></tr></table>';
-
-   document.getElementById('trade_live_chart').appendChild(div);
+  if (window.chartonly && window.chartonly === 'chartonly') {
```
Comment (qingweibinary) [inline src/javascript/binary/pages/trade/common.js]: what is chartonly variable used for? if it's only to indicate if it is chart only or not, it should be a boolean, if not, the name should be clearer for it's intention, (if possible)

Comment (negar-binary) [inline src/javascript/binary/pages/trade/common.js]: there are three different types of feed_license, one of them is chartonly, as is returned from websocket. I want to keep the same name as websocket to indicate it's using that value, but I can change it if you think it's needed. It ironically means that we don't have license to display the chart, so the opposite of chartonly. we discussed with quants, they said they may change the name in future.

Comment (qingweibinary) [inline src/javascript/binary/pages/trade/common.js]: I'd suggest we either store the license directly, eg. window.license = ???

or we store as flag, eg. 
```
window.chartAllowed = true,
window.ticksAllowed = false, etc
```

Comment (negar-binary) [inline src/javascript/binary/pages/trade/common.js]: sure, sounds good :) I'll make the change

**Pull request merged**